src/modules/70-pipeline/components/PipelineStudio/PipelineContext/PipelineContext.tsx

```diff
@@ -155,8 +155,9 @@ export const getPipelineByIdentifier = (
       obj = response
     }
     if (obj.status === 'SUCCESS' && obj.data?.yamlPipeline) {
+      const yamlPipelineDetails = parse(obj.data?.yamlPipeline)
       return {
-        ...parse(obj.data.yamlPipeline).pipeline,
+        ...(yamlPipelineDetails !== null && { ...yamlPipelineDetails.pipeline }),
```

Comment:
Please use ternary operator as incase of `true`, it will try to spread `true`, which internally is a `Boolean` object.
```suggestion
        ...(yamlPipelineDetails !== null ? { ...yamlPipelineDetails.pipeline }: {}),
```